### PR TITLE
feat: Add a Legal page to the Reference section

### PR DESCRIPTION
**Pull request by fghaas**
Create a page containing links to our legal documents.

Also, explain the relationship between the legacy GT&C and their replacement, the ToS.

### CODE REVIEW

File: docs/reference/legal/index.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+description: "The fine print: terms and agreements governing our services."
+---
+# Legal Reference
+
+This section contains links to the terms and agreements governing the {{brand}} services.
+
+## {{legal_docs.tos.name}}
+
+Our {{brand}} [{{legal_docs.tos.name}}]({{legal_docs.tos.url}}) are valid from 2023-04-05 if you created your account on or after that date.
```
Comment (colder-is-better) [inline docs/reference/legal/index.md:10]:
```suggestion
Our {{brand}} [{{legal_docs.tos.name}}]({{legal_docs.tos.url}}) are valid from 2023-04-05, provided you created your account on or after that date.
```

Comment (colder-is-better) [reply docs/reference/legal/index.md:10]: Looks more appropriate to me :)

File: docs/reference/legal/index.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+If you [created your account](../../howto/getting-started/create-account.md) before that date, the prior {{legal_docs.tc.name}} still apply until 2023-05-05, after which the {{legal_docs.tos.name}} supersede them.
+
+## {{legal_docs.tc.name}}
+
+The legacy [{{legal_docs.tc.name}}]({{legal_docs.tc.url}}) apply until 2023-05-05 if you created your account before 2023-04-05.
```
Comment (colder-is-better) [inline docs/reference/legal/index.md:16]: Same here:

```suggestion
The legacy [{{legal_docs.tc.name}}]({{legal_docs.tc.url}}) apply until 2023-05-05, provided you created your account before 2023-04-05.
```

**Review by colder-is-better — COMMENTED** [review]
I only have a couple of suggestions.

**Comment by fghaas** [comment]
> I only have a couple of suggestions.

Thanks, I've applied them both.

**Review by colder-is-better — APPROVED** [review]

**Review by armanborghem — APPROVED** [review]
Looks good!

**Pull request merged**